spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,16 +37,24 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile compatible implementation must pass *all* TCKs provided by each MicroProfile specification. Passing the
+Jakarta specifications TCKs is optional.
+
+These APIs and versions guarantee application developers a minimum set of APIs and are not intended to be restrictive.
```

Comment:
```suggestion
These APIs and versions provide application developers a minimum set of APIs, but they are not intended to be restrictive.
```

Reply:
I would keep guarantee as this meant as in guarantee portability for these APIs.